source/engineering/abaqus/single-node.md

```diff
@@ -0,0 +1,75 @@
+---
+title: "Single-Node Jobs"
+---
+
+Small scale Abaqus jobs can be run on a single node with up to 28 processors.
+
+## Job Script Template
+
+Here is an example job submission script for a single-node Abaqus job:
+
+```{code-block} bash
+---
+linenos: true
+---
+#!/usr/bin/env bash -l
```

Comment:
Good spot! I've come across this before, but it skipped my mind here while combining two sources of information!

Since these docs are targeted at specific systems, I'll just replace it with `/usr/bin/bash -l` which we know works.
I think relying on a login shell for bash scripts is poor practice, but I believe people in the BCI may be relying on some environment customization and I don't want to induce a number of hpc help queries based on differing environments.